# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/libxslt/hotfix-gcc14.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- libxslt-1.1.34/libxslt/extensions.c.vanilla	2024-02-27 17:00:15.998662751 +0100
+++ libxslt-1.1.34/libxslt/extensions.c	2024-02-27 17:00:21.323662482 +0100
@@ -14,6 +14,7 @@
 
 #include <string.h>
 #include <limits.h>
+#include <stdlib.h>
 
 #include <libxml/xmlmemory.h>
 #include <libxml/tree.h>
--- libxslt-1.1.34/libexslt/date.c.vanilla	2024-02-27 17:01:02.130660427 +0100
+++ libxslt-1.1.34/libexslt/date.c	2024-02-27 17:01:07.943660134 +0100
@@ -34,6 +34,8 @@
 #endif
 #endif
 
+#include <stdlib.h>
+
 #include <libxml/tree.h>
 #include <libxml/xpath.h>
 #include <libxml/xpathInternals.h>
